examples/01_Simulation.ipynb

```diff
@@ -1280,39 +1342,533 @@
+    "    'x': array([1.75, 1.8, 1.85, 1.9]),\n",
+    "    'v': array([35, 39, 22, 47])\n",
+    "}\n",
+    "\n",
+    "m = ThrownObject()\n",
+    "simulated_results = m.simulate_to_threshold(x=first_state, events='impact', event_strategy='all', print=True, dt=0.1, save_freq=1)\n",
+    "\n",
+    "print_vectorized_sim_plots(simulated_results)\n",
+    "plt.title(\"Vectorized simulation until any object hits the ground\", pad=10)\n",
+    "plt.show()"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Vectorized Simulation\n",
-    "**A version of this section will be added in release v1.8**"
+    "## Configuring Simulation"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Configuring Simulation\n",
-    "**A version of this notebook will be added in release v1.8, including:**\n",
-    "* t0, x\n",
-    "* integration_method"
+    "We can explore other ways to configure simulations. We will use the Battery model for these examples."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from progpy.models import BatteryCircuit as Battery\n",
+    "import matplotlib.pyplot as plt"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's create a model object and define a piecewise future loading function."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "batt = Battery()\n",
+    "\n",
+    "def future_loading(t, x=None):\n",
+    "    if (t < 600):\n",
+    "        i = 2\n",
+    "    elif (t < 900):\n",
+    "        i = 1\n",
+    "    elif (t < 1800):\n",
+    "        i = 4\n",
+    "    elif (t < 3000):\n",
+    "        i = 2     \n",
+    "    else:\n",
+    "        i = 3\n",
+    "    return batt.InputContainer({'i': i})"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Simulating From a Known Time"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "There may be cases where we want to simulate from a time other than 0. For example, we may have a future loading profile that is a function of time and need to pause our simulation midway (e.g., the results inform a decision) before continuing from where we left off.\n",
+    "\n",
+    "To do this, we can adjust `t0`. The following example shows a battery simulation that starts at 700 seconds."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "config = {\n",
+    "    'save_freq': 100,\n",
+    "    'dt': 2,\n",
+    "    't0': 700,\n",
+    "}\n",
+    "\n",
+    "simulated_results = batt.simulate_to_threshold(future_loading, **config)\n",
+    "\n",
+    "simulated_results.inputs.plot(xlabel='time (s)', ylabel='current draw (amps)')\n",
```

Comment:
This plot is somewhat useful, since it does show that beginning time starts at 700sec, but I'm wondering if there's something else we can do to show this more clearly. Any ideas? One idea might be to print out the first time point in the results, which should presumably be about 700sec. If we keep a plot, perhaps a voltage curve would make more sense than the current/input curve?

Reply:
A voltage curve makes sense and we could print out the first timestamp of the simulation results. Will see if Chris has any other thoughts on this.

Reply:
Updated - added printed first timestamp, annotated first data point, and both input and voltage graphs.